rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/370_profile.yml

```diff
@@ -21,8 +21,8 @@ setup:
 
 ---
 fetch fields:
-  - skip:
-      version: ' - 8.14.99'
+  - requires:
+      cluster_features: "gte_v8.15.0"
```

Comment:
Have we spoken to the author to point them towards the newer way?

Reply:
Is it also worth failing if `version` is specified >=8.15?

Reply:
> Have we spoken to the author to point them towards the newer way?

I haven't, happy to follow up once we have to documentation ready to be shared.

> Is it also worth failing if version is specified >=8.15?

Attempting to use `gte_v8.15.1` would fail the test. Previously that was already the case for `gte_v8.14.1`, but I had to expand it for this PR. The only way forward is removing `version` asap, and have appropriate alternatives in place (including test-only cluster features).